modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -34,20 +35,14 @@ data:
     backend:
       auth:
         keys:
-          - secret: "${BACKEND_SECRET}"
-      baseUrl: https://backstage-<custom_resource_name>-<namespace_name>.<openshift_ingress_domain>
+          - secret: "${BACKEND_SECRET}" # <1>
+      baseUrl: <{product-very-short}_URL> # <2>
```

Comment:
Should we also replace the `app.baseUrl` field (L34) with `<{product-very-short}_URL>`?

Reply:
Indeed!

Reply:
Done